docs/content/latest/releases/whats-new/_index.md

```diff
@@ -40,38 +40,95 @@ menu:
+- Change intent iterator creation logic. [#4543](https://github.com/yugabyte/yugabyte-installation/pull/4543)
+- Upgrade all Python scripts used to build and package the code to Python 3. [#1442](https://github.com/yugabyte/yugabyte-db/issues/1442)
 
 ## Yugabyte Platform
 
-- When performing a full move or add node on a universe that has a yb-master, the `server.conf` file is now being updated with the new `master_addresses`. [#4242](https://github.com/yugabyte/yugabyte-db/issues/4242)
-- In the **Backups** tab, individual YSQL tables can no longer be selected. Previously, attempting to back up a YSQL table would create a failed task. [#3848](https://github.com/yugabyte/yugabyte-db/issues/3848)
-- In the **Metrics** view, transactions have been added to the YSQL and YCQL operations charts. [#3827](https://github.com/yugabyte/yugabyte-db/issues/3827)
-- **Create Read Replica** and **Edit Read Replica** pages are no longer in beta. [#4313](https://github.com/yugabyte/yugabyte-db/issues/4313)
-- In the **Certificates** page, you can now download certificates. [#3985](https://github.com/yugabyte/yugabyte-db/issues/3985)
-- In the **Universes** overview page, add a button to toggle on metrics graph widgets to auto-refresh or to set refresh interval. [#2296](https://github.com/yugabyte/yugabyte-db/issues/2296)
+- Fix failure when adding a node on a TLS-enabled universe. [#4482](https://github.com/yugabyte/yugabyte-db/issues/4482)
+- Improve latency tracking by splitting overall operation metrics into individual rows for each API. [#3825](https://github.com/yugabyte/yugabyte-db/issues/3825)
+  - YCQL and YEDIS metrics include `ops`, `avg latency`, and `P99 latency`.
+  - YSQL metrics include only `ops` and `avg latency`.
+- Add metrics for RPC queue sizes of services, including YB-Master, YB-TServer, YCQL, and YEDIS. [#4294](https://github.com/yugabyte/yugabyte-db/issues/4294)
+- Add option to edit configuration flags without requiring server restart. [#4433](https://github.com/yugabyte/yugabyte-db/issues/4433)
+- When configuration flags are deleted in the YugabyteDB Admin Console, remove the flags from `server.conf` file. [#4341](https://github.com/yugabyte/yugabyte-db/issues/4341)
+- When creating GCP instances, only use host project when specifying network.
+- When creating a cloud provider configuration, display provider-level (non-k8s) settings for SSH ports and enabling airgapped installations. [#3615](https://github.com/yugabyte/yugabyte-db/issues/3615), [#4243](https://github.com/yugabyte/yugabyte-db/issues/4243), and [#4240](https://github.com/yugabyte/yugabyte-db/issues/4240).
+- After removing a node and then adding a node, check for certificate and key files and create the files if needed. [#4551](https://github.com/yugabyte/yugabyte-db/issues/4551)
+- Update to support Helm 3 deployments. [#4416]((https://github.com/yugabyte/yugabyte-db/issues/4416))
```

Comment:
it should also say that Helm 2 is no longer supported and should link to the doc where we show how to migrate an existing Helm 2 universe to Helm 3.